README.md

```diff
@@ -246,10 +238,13 @@ requiring in chai-http. For example:
 ```js
 // Add promise support if this does not exist natively.
 if (!global.Promise) {
-  global.Promise = require('q');
+  global.Promise = await import('q');
```

Comment:
we can probably just drop this part of the readme (this whole block about promises) since all platforms have had them for some time now